geometry/instant_output_test.cpp

```diff
@@ -0,0 +1,141 @@
+﻿
+#include <strstream>
+
+#include "astronomy/time_scales.hpp"
+#include "geometry/named_quantities.hpp"
+#include "geometry/sign.hpp"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+namespace principia {
+namespace geometry {
+namespace internal_point {
+
+using astronomy::InfiniteFuture;
+using astronomy::InfinitePast;
+using astronomy::J2000;
+using astronomy::operator""_TT;
+using astronomy::operator""_UT1;
+using astronomy::operator""_UTC;
+using quantities::Time;
+using quantities::si::Second;
+using ::testing::Eq;
+
+class InstantOutputTest : public ::testing::Test {
+ protected:
+  Instant JustAfter(Instant const t) {
+    return J2000 + std::bit_cast<double>(
```

Comment:
Could this use `std::nextafter`?

Reply:
Yes; I need to implement a constexpr IEEE 754 nextUp/nextDown to fix the clang build anyway.